bundles/org.openhab.automation.jrubyscripting/README.md

```diff
@@ -1210,6 +1229,29 @@ rules[rule_uid].enable
 rules[rule_uid].disable
 ```
 
+#### Passing Values to Rules
+
+A rule/script may be given additional context/data by the caller. This additional data is available
+within the rule by referring to the names of the context variable. This is applicable to both
+UI rules and file-based rules.
```

Comment:
Generally, if you put each sentence on a separate line, it will be easier to view diffs (going forward):
```suggestion
A rule/script may be given additional context/data by the caller.
This additional data is available within the rule by referring to the names of the context variable.
This is applicable to both UI rules and file-based rules.
```

Reply:
I've made some minor changes to the source of this file. Not _all_ sentences are on their own line. Some shorter sentences share a line as long as they are not broken up. And some longer sentences are split up so they wrap.
This will hopefully make reviewing changes easier.

https://github.com/openhab/openhab-jruby/pull/169

Once that's merged, I'll then have to regenerate the addon-readme and push into this PR.

Would that be OK, or do you still require strict one line per sentence regardless of their length?

Reply:
> Some shorter sentences share a line as long as they are not broken up.

I would recommend against that for the same reason as my original comment. 🙂

> And some longer sentences are split up so they wrap.

I understand the dilemma here, but probably it would be best to keep even long sentences in one line. This will also make it visible if some sentences are actually too long, and could be divided/rephrased.

Imagine you are editing the beginning of such a long sentence which is already wrapped into multiple lines. Now you are making it even longer, so you might want to wrap it differently. You will now end up with a hard to read diff, only because some contents were moved from first line to second line because of adjusted wrapping. This is what we want to prevent by having separate lines for each sentence.

> [openhab/openhab-jruby#169](https://github.com/openhab/openhab-jruby/pull/169)
> 
> Once that's merged, I'll then have to regenerate the addon-readme and push into this PR.
> 
> Would that be OK, or do you still require strict one line per sentence regardless of their length?

This is perfectly OK, it's not a strict rule, just a recommendation.

Reply:
I've updated the markdown following the recommendation.